cmds/core/head/head.go

```diff
@@ -36,40 +36,68 @@ func run(stdout io.Writer, stderr io.Writer, bytes, count int, files ...string)
 	var newLineHeader bool
 	var errs error
 
-	for _, file := range files {
-		f, err := os.Open(file)
-		if err != nil {
-			errs = errors.Join(errs, fmt.Errorf("head: %w", err))
-			continue
-		}
+	var handle = func(r io.Reader, name string) error {
 		if len(files) > 1 {
 			if newLineHeader {
-				fmt.Fprintf(stdout, "\n==> %s <==\n", filepath.Base(f.Name()))
+				fmt.Fprintf(stdout, "\n==> %s <==\n", filepath.Base(name))
```

Comment:
If I do this:
head /etc/passwd /tmp/passwd
Don't I want to see the full path? I'm just wondering about reducing to filepath.Base?

Reply:
done